src/services/dashboards/all-dashboards/modules/AllDashboardsSelectFilter.vue

```diff
@@ -84,6 +85,13 @@ export default defineComponent<AllDashboardsSelectFilterProps>({
             proxyScopeStatus: useProxyValue('scopeStatus', props, emit),
         });
 
+        watch(() => state.proxyViewersStatus, (selectedViewers) => {
+            store.dispatch('dashboard/setSelectedViewers', selectedViewers);
+        });
+        watch(() => state.proxyScopeStatus, (selectedScope) => {
+            store.dispatch('dashboard/setSelectedScope', selectedScope);
+        });
```

Comment:
question: The reason why viewerStatus, scopeStatus props exist is to show data flow?